### ISSUE TITLE
How to evaluate results?

### ISSUE THREAD

**Issue by ktnguyen2** [Vladkryvoruchko/PSPNet-Keras-tensorflow#61]
I was wondering how to get mean IoU and pixel accuracy?

**Comment by ga84** [comment]
I am also interested in this. Would it be possible for you to upload some script with evaluation code, or is it too much to ask? :) @Vladkryvoruchko  @jmtatsch
